src/AbstractOperations/computed_field.jl

```diff
@@ -89,3 +89,62 @@ end
 
     @inbounds data[i′, j′, k′] = operand[i′, j′, k′]
 end
+
+struct FusedComputedFields
+    fields::Vector{ComputedField}
```

Comment:
I think we need this to be either `Tuple` or `NamedTuple`. Probably the best thing is to leave it as an abstract type so we can use either Tuple or NamedTuple as needed (`NamedTuple` may be useful for output)